jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -0,0 +1,126 @@
+{{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool }}
+{{ $ENABLE_JIBRI_SIP := .Env.ENABLE_JIBRI_SIP | default "0" | toBool }}
+
+jicofo {
+    {{ if $ENABLE_AUTH }}  
+    authentication {
+      enabled = true
+      // The type of authentication. Supported values are XMPP, JWT or SHIBBOLETH (default).
+      {{ if eq $AUTH_TYPE "jwt" }}
+      type = JWT
+      {{ else }}
+      type = XMPP
+      {{ end }}
+      login-url = "{{ .Env.XMPP_DOMAIN }}"
+    }
+    {{ end }}
+
+    // Configuration related to jitsi-videobridge
+    bridge {
+      {{ if .Env.MAX_BRIDGE_PARTICIPANTS }}
+      max-bridge-participants = "{{ .Env.MAX_BRIDGE_PARTICIPANTS }}"
+      {{ end }}
+      {{ if .Env.OCTO_BRIDGE_SELECTION_STRATEGY }}
+      selection-strategy = "{{ .Env.OCTO_BRIDGE_SELECTION_STRATEGY }}"
+      {{ end }}
+
+      {{ if .Env.JICOFO_ENABLE_BRIDGE_HEALTH_CHECKS }}
+      health-checks {
+        enabled = "{{ .Env.JICOFO_ENABLE_BRIDGE_HEALTH_CHECKS }}"
+      }
+      {{ end }}
+  
+      brewery-jid = "{{ .Env.JVB_BREWERY_MUC }}@{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}"
+    }
+    // Configure the codecs and RTP extensions to be used in the offer sent to clients.
+    codec {
+      video {
+        {{ if .Env.ENABLE_CODEC_VP8 }}
+        vp8 {
+          enabled = "{{ .Env.ENABLE_CODEC_VP8 }}"
+        }
+        {{ end }}
+        {{ if .Env.ENABLE_CODEC_VP9 }}
+        vp9 {
+          enabled = "{{ .Env.ENABLE_CODEC_VP9 }}"
+        }
+        {{ end }}
+        {{ if .Env.ENABLE_CODEC_H264 }}
+        h264 {
+          enabled = "{{ .Env.ENABLE_CODEC_H264 }}"
+        }
+        {{ end }}
+      }
+    }
+  
+    conference {
+      {{ if .Env.ENABLE_AUTO_OWNER }}
+      enable-auto-owner = "{{ .Env.ENABLE_AUTO_OWNER }}"
+      {{ end }}
+
+      {{ if .Env.JICOFO_CONF_INITIAL_PARTICIPANT_WAIT_TIMEOUT }}
+      initial-timeout = "{{ .Env.JICOFO_CONF_INITIAL_PARTICIPANT_WAIT_TIMEOUT }}"
+      {{ end }}
+
+      {{ if .Env.JICOFO_CONF_SINGLE_PARTICIPANT_TIMEOUT }}
+      single-participant-timeout = "{{ .Env.JICOFO_CONF_SINGLE_PARTICIPANT_TIMEOUT }}"
+      {{ end }}
+    }
+  
+    {{ if .Env.JICOFO_ENABLE_HEALTH_CHECKS }}
+    // Configuration for the internal health checks performed by jicofo.
+    health {
+      // Whether to perform health checks.
+      enabled = "{{ .Env.JICOFO_ENABLE_HEALTH_CHECKS }}"
+    }
+    {{ end }}
+  
+    {{ if $ENABLE_RECORDING }}
+    jibri {
+      brewery-jid = "{{ .Env.JIBRI_BREWERY_MUC}}@{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}"
+      {{ if .Env.JIBRI_REQUEST_RETRIES }}
+      num-retries = "{{ .Env.JIBRI_REQUEST_RETRIES }}"
+      {{ end }}
+      {{ if .Env.JIBRI_PENDING_TIMEOUT }}
+      pending-timeout = "{{ .Env.JIBRI_PENDING_TIMEOUT }}"
+      {{ end }}
+    }
+    {{ end }}
+  
+    {{ if $ENABLE_JIBRI_SIP }}
+    jibri-sip {
+      // The JID of the MUC to be used as a brewery for jibri instances for SIP.
+      # brewery-jid = "jibrisipbrewery@example.com"
```

Comment:
@saghul @sapkra  Can you please help here? What is the value that I need to put in here? Is there any env variable that I can refer to for `jibrisipbrewery` ?
`brewery-jid = "jibrisipbrewery@{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}"`

Reply:
I'd drop the Jibri SIP config part. It's not yet ready for prime time.